### PR TITLE
Hogeun Adds Seoul File

### PR DESCRIPTION
**Pull request by hogeuns**
This PR adds content to the Seoul :mouse:  :roller_coaster: 

Some open :question: for this PR:
- if you've never visitsed Seoul -- what kind of things interest you?
- @crichID what kind of things do you like to do with younglings?

### CODE REVIEW

**Comment by robden12** [comment]
ship it! :shipit:

File: hogeuns-seoul.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Seoul, Korea
+
+## Restaurants
+
```
Comment (robden12) [inline hogeuns-seoul.md:5]: Where is the best Kimchi?

Comment (githubteacher) [inline hogeuns-seoul.md:5]: @hogeuns are there any good Korean BBQ places? 🇰🇷

Comment (hogeuns) [inline hogeuns-seoul.md:5]: Try Seoul Garden

**Comment by hogeuns** [comment]
@CSAmador I added some Gator stuff in 7e70342

**Comment by hogeuns** [comment]
@crichID does this look ready to :ship:

**Pull request merged**